src/Uno.UI/UI/Xaml/DependencyProperty.cs

```diff
@@ -54,7 +54,7 @@ private DependencyProperty(string name, Type propertyType, Type ownerType, Prope
 		{
 			_name = name;
 			_propertyType = propertyType;
-			_ownerType = attached || IsTypeDependencyObject(ownerType) ? ownerType : typeof(_View);
+			_ownerType = attached || IsTypeDependencyObject(ownerType) ? ownerType : ownerType;
```

Comment:
Codacy found an issue: [This conditional operation returns the same value whether the condition is "true" or "false".](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6166183)